test/functional/fixtures/tty-test.c

```diff
@@ -34,6 +34,18 @@ bool owns_tty(void)
 #endif
 }
 
+static inline bool is_terminal(FILE *stream1)
+  REAL_FATTR_ALWAYS_INLINE REAL_FATTR_PURE REAL_FATTR_WARN_UNUSED_RESULT;
+
+
+ // Checks if the file pointer is not a terminal like STDIN or STDOUT
+ // @param[in]  stream  to be checked
```

Comment:
`stream` is not part of a sentence. Missing dot and capital letter at the start. Not really needed, so just remove the line.